lib/pulselistener.js

```diff
@@ -51,12 +52,29 @@ var retryConnect = function(connectionString, retries) {
  * {
  *   namespace:         // Namespace to prefix queues/exchanges (optional)
  *                      // defaults to `username` if given otherwise ""
- *   username:          // Username to connect with (and namespace if not given)
- *   password:          // Password to connect with
- *   hostname:          // Hostname to connect to using username/password
- *                      // defaults to pulse.mozilla.org
- *   connectionString:  // connectionString cannot be used with username,
- *                      // password and/or hostname.
+ *   hostname:          // Hostname to connect to (optional, defaults to
+ *                      // pulse.mozilla.org)
+ *   // Contact info sent to taskcluster-pulse when requesting credentials
+ *   // (required when credentials are managed by taskcluster-pulse)
+ *   contact: {         // One of:
```

Comment:
Could we default this to:
```js
contact: {
  method: 'email',
  payload: {
    address: process.env.OWNER_EMAIL,
  },
}
```

We currently have a convention that we set the environment variable `OWNER_EMAIL` so we can easily see who owns a service...

If you rather call the env var something else we can also rename it...

@imbstack, thoughts?